depends/toolchain.cmake.in

```diff
@@ -86,7 +86,11 @@ set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
 set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
-set(QT_TRANSLATIONS_DIR "${CMAKE_CURRENT_LIST_DIR}/translations")
+if(@depends_crosscompiling@)
```

Comment:
Can we move the translations into `/translations` and then remove the need for this condition?

Reply:
Thanks! [Fixed](https://github.com/bitcoin/bitcoin/pull/30997#issuecomment-2740271781).